### ISSUE TITLE
Failed downloading the Cypress 7.3.0 binary 

### ISSUE THREAD

**Issue by fredrikgiang** [cypress-io/cypress#16440]
### Current behavior
We are experiencing intermittent connectivity issues when attempting to install the binary on a Linux.

URL: https://download.cypress.io/desktop/7.3.0?platform=linux&arch=x64
Error: Failed downloading the Cypress binary.
Response code: 404
Response message: Not Found

### Download method
Use yarn to install

### Operating System
Platform: linux (Debian - 10.8)
Cypress Version: 7.3.0

**Comment by chenxsan** [comment]
@tgriesser I'm running into the issue again, see https://github.com/webpack/webpack.js.org/pull/4976/checks?check_run_id=2575406294

**Comment by liaokaime** [comment]
I cannot download it. too. My Response code is 403. 

```yarn add cypress```

then

```
URL: https://download.cypress.io/desktop/7.3.0?platform=win32&arch=x64
Error: Failed downloading the Cypress binary.
Response code: 403
Response message: Forbidden
```

But if I copy this link to the browser, it can be downloaded

`https://download.cypress.io/desktop/7.3.0?platform=win32&arch=x64`

**Comment by jennifer-shehane** [comment]
@liaokaime I don't think this is related to the original issue since thee response is different. Could you [open a new issue](https://github.com/cypress-io/cypress/issues/new?assignees=&labels=topic%3A+installation&template=2-install-issue.md&title=)
